Scheduler/observing_blocks_maintel/BLOCK-197.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "BLOCK-197",
+    "program": "BLOCK-197",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "run_command.py",
+            "standard": true,
+            "parameters": {
+                "component": "MTHexapod:1",
+                "cmd": "offset",
+                "parameters": {
+                    "y": 500
+                }
+            }
+        },
+        {
+            "name": "maintel/laser_tracker/align.py",
+            "standard": true,
+            "parameters": {
+                "target": "M2",
+                "tolerance_linear": 0.02,
```

Comment:
Just an FYI, Scott and I found that the measurement rms is around 0.3mm, I think we might need to increase this tolerance here or the script might fail.